CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ option(ENABLE_BENCHMARKS "Build benchmarks" Off)
 option(RAJA_DEPRECATED_TESTS "Test deprecated features" Off)
 option(RAJA_ENABLE_BOUNDS_CHECK "Enable bounds checking in RAJA::Views/Layouts" Off)
 option(RAJA_TEST_EXHAUSTIVE "Build RAJA exhaustive tests" Off)
+option(ENABLE_RUNTIME_PLUGINS "Enable support for loading plugins at runtime" Off)
```

Comment:
I'm happy leaving these off by default, but we probably want to make sure we run some of the travis tests with these on

Reply:
Agreed. @davidbeckingsale we have an issue to clean up our CMake variables by prepending them with 'RAJA_' to make it clear which ones we own. Will you please change the name of this one following that?

Reply:
Should be all fixed up now.